digamma.go

```diff
@@ -0,0 +1,26 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// Digamma returns the logorithmic derivative of the gamma function at x.
+//  ψ(x) = d/dx (Ln (Γ(x)).
+func Digamma(x float64) float64 {
+	// This is adapted from
```

Comment:
It looks like this code doesn't work for integers between -7 and 0. The comment should include what it doesn't work for, and there should be a panic with it.